renderdoc/driver/metal/metal_helpers_bridge.mm

```diff
@@ -104,3 +104,19 @@
   CA::MetalDrawable *drawable = (__bridge CA::MetalDrawable *)[metalLayer nextDrawable];
   return drawable;
 }
+
+@interface ObjCTrackedCAMetalLayer : NSObject
```

Comment:
In the future, if we end up needing to track more ObjC object's lifetime then I might make this a more general class or use the macro approach similar to for the wrapped ObjC objects.